### PR TITLE
Fix tests for develop stabilization

### PR DESCRIPTION
**Pull request by Kozoriz**
Corrected SQL pt representation tests.
Removed one useless test in Timer (tests functionality that removed)
Disabled some tests (due resolving timer defect).

### CODE REVIEW

**Comment by Kozoriz** [comment]
Please review : @anosach-luxoft , @LuxoftAKutsan

File: src/components/utils/test/timer_test.cc [hunk 1]
```diff
@@ -151,47 +141,61 @@ TEST_F(TimerTest, Start_Loop_3Calls) {
...
+  timeout_=1000u;
+  EXPECT_CALL(*test_task_, run()).Times(0);
   // Actions
   test_timer.Start(timeout_, single_shot_);
   // Expects
   ASSERT_EQ(timeout_, test_timer.timeout());
   ASSERT_TRUE(test_timer.is_running());
   // Actions
-  timeout_ = 1000u;
+  timeout_ = 200000u;
```
Comment (LuxoftAKutsan) [inline src/components/utils/test/timer_test.cc]: @Kozoriz  increasing timeout fixes tests? Looks strange

Comment (Kozoriz) [inline src/components/utils/test/timer_test.cc]: @LuxoftAKutsan , this is not fix for tests. This test disabled by me. Timeout changed for easier reproduce of problem in timer. If we start and stop immediately, timer not stopped and going to timeout. With this change we can call pause in debugger and see whats wrong.

Comment (LuxoftAKutsan) [inline src/components/utils/test/timer_test.cc]: @Kozoriz  Please remove this increasing of timeout it is very confusing!

Comment (Kozoriz) [inline src/components/utils/test/timer_test.cc]: @LuxoftAKutsan , why? we don`t care how big timeout can be putted to timer. And with this timeout we can easer find problem.

**Comment by LuxoftAKutsan** [comment]
Reviewed

File: src/components/utils/test/timer_test.cc (hunk 1 above)
Comment (Kozoriz) [inline src/components/utils/test/timer_test.cc]: @LuxoftAKutsan , done in 
	7404b6f

**Pull request merged**